source/prescribed_stokes_solution/circle.cc

```diff
@@ -0,0 +1,59 @@
+/*
+  Copyright (C) 2011 - 2014 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file doc/COPYING.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#include <aspect/global.h>
+#include <aspect/prescribed_stokes_solution/interface.h>
+
+#include <deal.II/base/exceptions.h>
+#include <deal.II/base/std_cxx1x/tuple.h>
+
+#include <list>
+namespace aspect
+{
+  namespace PrescribedStokesSolution
+  {
+    template <int dim>
+    class Circle: public Interface<dim>
+      {
+      public:
+
+    	virtual
+    	void   solution (const Point<dim>   &p, Vector<double>     &value) const
+    	{
+    		value(0) = -p(1);
+    		value(1) = p(0);
+    		value(dim) = 0;
+    	}
+};
+}
+}
+
+// explicit instantiations
+namespace aspect
+{
+  namespace PrescribedStokesSolution
+  {
+    ASPECT_REGISTER_PRESCRIBED_STOKES_SOLUTION(Circle,
+                                       "circle",
+                                       "Zero pressure and velocity rotating around origin")
```

Comment:
@bangerth: I am not sure, should this description be a sentence, or are notes sufficient?